fuzz/src/chanmon_consistency.rs

```diff
@@ -1116,12 +1132,16 @@ pub fn do_test<Out: test_logger::Output>(data: &[u8], out: Out) {
 				}
 
 				// Finally, make sure that at least one end of each channel can make a substantial payment.
-				assert!(
-					send_payment(&nodes[0], &nodes[1], chan_a, 10_000_000, &mut payment_id) ||
-					send_payment(&nodes[1], &nodes[0], chan_a, 10_000_000, &mut payment_id));
-				assert!(
-					send_payment(&nodes[1], &nodes[2], chan_b, 10_000_000, &mut payment_id) ||
-					send_payment(&nodes[2], &nodes[1], chan_b, 10_000_000, &mut payment_id));
+				if out.may_fail.load(atomic::Ordering::Acquire) {
```

Comment:
Actually, thinking about it more, I don't think we should have this here at all? One side of the channel should *always* be able to send an HTLC after all pending HTLCs have been removed, no matter the fee state. This indicates a channel-stuck condition, not a channel-failure condition.